OutOfSchool/OutOfSchool.WebApi.Tests/Services/Database/WorkshopServiceTests.cs

```diff
@@ -672,14 +672,46 @@ private void SetupCreate()
             .Returns(Task.FromResult(provider));
         workshopRepository.Setup(
                 w => w.Create(It.IsAny<Workshop>()))
-            .Returns(Task.FromResult(It.IsAny<Workshop>()));
+            .ReturnsAsync((Workshop workshop) =>
+            {
+                return new Workshop
+                {
+                    Id = new Guid("8f91783d-a68f-41fa-9ded-d879f187a94e"),
+                    AvailableSeats = workshop.AvailableSeats,
+                };
+            });
         workshopRepository.Setup(
                 w => w.RunInTransaction(It.IsAny<Func<Task<Workshop>>>()))
-            .ReturnsAsync(new Workshop() { Id = new Guid("8f91783d-a68f-41fa-9ded-d879f187a94e") });
+            .ReturnsAsync((Func<Task<Workshop>> func) =>
+            {
+                return new Workshop
+                {
+                    Id = new Guid("8f91783d-a68f-41fa-9ded-d879f187a94e"),
+                    AvailableSeats = func.Invoke().Result.AvailableSeats,
```

Comment:
(Code readability)  
1. I believe that `func.Invoke()` could be rewritten to just `func()`.
2. Not sure, but probably ssync part with `.Result` could be better highlighted using `.Returns(async ` vs your current `.ReturnsAsync( .... /* and lot later we have */ .Result. ....)`, see below:
```
... .Returns(async (Func<Task<Workshop>> func) =>
{
    var workshop = await func();
    return new Workshop
    {
        Id = new Guid("8f91783d-a68f-41fa-9ded-d879f187a94e"),
        AvailableSeats = workshop.AvailableSeats; 
...
```
In any case please avoid usage of `.Result`